### ISSUE TITLE
#-1461309901------------- Heading levels should only increase by one

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1860]
<strong>PageName:</strong> FeedViewDetailsPage  

<strong>Source:</strong>`<h4 class="leader-quarter">Description</h4>`

<strong>Element Location: </strong>.trailer-2[data-dojo-attach-point="_textView"] > div[data-dojo-attach-point="containerNode"] > .leader-quarter

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Heading order invalid

<strong>Tags:</strong> cat.semantics,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/heading-order?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedViewDetailsPage\-1461309901_Heading levels should only increase by one.png`
